src/components/record/related-table-actions.tsx

```diff
@@ -496,50 +603,293 @@ const RelatedTableActions = ({
+        buttons = Array.from(buttonContainer.getElementsByClassName('chaise-btn'));
+
+        const tableHeaderButtonsToAddToDropdown: any = [];
+        let buttonLeftOffset = 0;
+        let totalWidth = 0;
+
+        if (isMobile) {
+          setButtonsInDropdown(buttons);
+          buttonContainer.style.display = 'none';
+          return;
+        } else {
+          buttonContainer.style.display = 'block';
+          setButtonsInDropdown([]);
+          buttons.forEach((button: any, index: number) => {
+            const buttonWidth = button.getBoundingClientRect().width;
+            if (index === 0) {
+              buttonLeftOffset =
+                button.getBoundingClientRect().left - buttonContainer.getBoundingClientRect().left;
+            }
+
+            if (totalWidth + buttonLeftOffset + buttonWidth <= containerWidth) {
+              totalWidth += buttonWidth;
+            } else {
+              tableHeaderButtonsToAddToDropdown.push(button);
+            }
+          });
+          setButtonsInDropdown(tableHeaderButtonsToAddToDropdown);
+        }
+      }
+    };
+    calculateButtons();
+
+    const mainNavResizeSensor = new ResizeSensor(mainnav as Element, () => {
+      calculateButtons();
+    });
+
+    return () => {
+      mainNavResizeSensor.detach();
+    };
+  }, [isMobile]);
+
+  const toggleDropdown = (bool: boolean, evt: any) => {
+    evt.originalEvent.stopPropagation();
+    setIsDropdownOpen(!isDropdownOpen);
+  };
+  const containerClassName = `related-table-actions ${isMobile ? 'is-mobile ' : ''}${
+    istableHeader ? 'table-header-actions' : 'main-section-actions'
+  }`;
+
+  /*
+   * Callback function for dropdown item click
+   */
+  const handleDropDownClick = (evt: any, button: any) => {
+    const buttonText = button.textContent;
+    evt.stopPropagation();
+    switch (buttonText) {
+      case 'Explore':
+        onExplore(evt);
+        break;
+      case 'Link Records':
+      case 'Add Records':
+        onCreate(evt);
+        break;
+      case 'Edit Records':
+        onUnlink(evt);
+        break;
+      case 'Unlink records':
+        onUnlink(evt);
+        break;
+      case 'Edit mode':
+      case 'Custom mode':
+      case 'Table mode':
+        onToggleDisplayMode(evt);
+        break;
+    }
+  };
+  return (
+    <>
+      <div className={containerClassName} ref={containerRef}>
+        {renderButtons()}
+        {buttonsInDropdown.length > 0 && (
+          <Dropdown onToggle={(isOpen: boolean, event: any) => toggleDropdown(isOpen, event)}>
+            <Dropdown.Toggle
+              disabled={false}
+              className='chaise-btn chaise-btn-primary dropdown-toggle-table'
+            >
+              <span className='fa fa-angle-double-right'></span>
+            </Dropdown.Toggle>
+            <Dropdown.Menu>
+              {buttonsInDropdown.map((button: any, index) => (
+                <Dropdown.Item
+                  className={''}
```

Comment:
Add `as={'li'}` here so it doesn't wrap elements inside a link.

```
<Dropdown.Item
  key={`dropdown-button-${index}`}
  as={'li'}
  onClick={(e: any) => handleDropDownClick(e, button)}
/>
```

Reply:
Please make sure there aren't any errors both in dev and production build mode. Because of this, React was complaining about usage of `a` tag inside another `a` tag.